index.md

```diff
@@ -64,11 +64,11 @@ Often it is best when you first contact an organization to assume that they don
 
 ### Asking Others to Help {#ask}
 
-Consider asking for help understanding the problem and communicating it. Someone might be able to help you understand the website or assistive technologies. Someone might help you communicate to the website owners. For example, an advocacy group.
+Consider asking friends, family, or colleagues for help understanding the problem you are observing and communicating it to the website owner. For example, the issue may be related to the website or any assistive technologies you are using. You can also ask other to help you communicate with the website owner. For example, an advocacy group.
 
 ### Encouraging Accessible Websites {#encourage}
 
-Acknowledge and encourage organizations that make their websites accessible. Positive feedback can motivate individuals and organizations. It can encourage other organizations to invest in web accessibility. 
+Acknowledge and encourage organizations that make an effort to be accessible. Positive feedback can motivate individuals and organizations. It can encourage other organizations to invest in web accessibility. 
```

Comment:
My mind drifted off wondering how many organizations have accessible websites, and what encouragement (vs praise) they would need.